### PR TITLE
[Digital Ocean] DO-7442  upgrade godo client to latest version

### PR DESCRIPTION
**Pull request by srikiz**
Currently Digital Ocean's godo client is referrring to a very old dependency version (1.1.3). Upgrading the same to the latest release tag (v1.19.0) commit.
This is needed to utilize the tags when creating a volume request. This PR only addresses the Go Dependencies, a different PR will be created to handle the logic around tags.

### CODE REVIEW

File: pkg/resources/digitalocean/dns/dns.go [hunk 1]
```diff
@@ -17,15 +17,14 @@ limitations under the License.
 package dns
 
 import (
+	"context"
 	"errors"
 	"fmt"
+	"github.com/digitalocean/godo"
```
Comment (zetaab) [inline pkg/resources/digitalocean/dns/dns.go]: style note: usually when I am coding golang, the internal libraries are imported first. Then rowchange and after that other imports. Like in this case I recommend moving godo after go internal imports

Comment (srikiz) [reply pkg/resources/digitalocean/dns/dns.go]: Sure, updated.

**Comment by srikiz** [comment]
FYI - @timoreimann please have a look when you get a chance. This is for upgrading godo dependencies to the latest version.

**Comment by justinsb** [comment]
Thanks @srikiz!

/approve
/lgtm

**Pull request merged**